oci/layout/oci_dest_test.go

```diff
@@ -105,7 +106,31 @@ func TestPutManifestTwice(t *testing.T) {
 
 	index, err := ociRef.getIndex()
 	assert.NoError(t, err)
-	assert.Equal(t, 2, len(index.Manifests), "Unexpected number of manifests")
+	assert.Len(t, index.Manifests, 2, "Unexpected number of manifests")
+}
+
+func TestPutTwoDifferentTags(t *testing.T) {
+	ref, tmpDir := refToTempOCI(t)
+	defer os.RemoveAll(tmpDir)
+
+	ociRef, ok := ref.(ociReference)
+	require.True(t, ok)
+
+	putTestConfig(t, ociRef, tmpDir)
+	putTestManifest(t, ociRef, tmpDir)
+
+	// add the same manifest with a different tag; it shouldn't get overwritten
+	ref, err := NewReference(tmpDir, "zomg")
+	ociRef, ok = ref.(ociReference)
+	require.True(t, ok)
+	assert.NoError(t, err)
+	putTestManifest(t, ociRef, tmpDir)
```

Comment:
```suggestion
	ref, err := NewReference(tmpDir, "zomg")
	require.NoError(t, err)
	ociRef, ok = ref.(ociReference)
	require.True(t, ok)
	putTestManifest(t, ociRef, tmpDir)
```
or the error check would probably not trigger.

Reply:
Oh, yeah, fixed thanks.